frontend/packages/noobaa-storage-plugin/src/components/data-resiliency-card/data-resiliency-card.scss

```diff
@@ -13,23 +10,28 @@
 }
 
 .nb-data-resiliency__icon--error {
-  color: $pf-color-red-100;
+  font-size: 60px;
 }
 
 .nb-data-resiliency__icon--ok {
-  color: $pf-color-light-green-400;
+  font-size: 60px;
 }
 
 .nb-data-resiliency__status-title--error {
+  font-size: 0.875rem;
   margin-top: 1.5625em;
 }
 
 .nb-data-resiliency__status-title--ok {
-  margin-bottom: 3.5em;
+  font-size: 0.9rem;
+  margin-top: 1em;
+  margin-bottom: 3em;
 }
 
 .nb-data-resiliency__title {
-  margin-bottom: 6em;
+  font-size: 1rem;
```

Comment:
why different font sizes for `title`, `title--ok` and `title--error`. Shouldn't it be the same ?

Reply:
Agree to you, the reasons being:
`title` : the text was small compared to others card
`title--ok` : the text was too small but don't want it to be that big as `title`

Reply:
I find this card inconsistent. If everything is ok we have text below icon, if there's an error, we have text above icon. Shouldn't it be unified ? @andybraren

Reply:
@rawagner I conveyed the same in team.
@a2batic is taking care of inconsistencies of dashboards regarding unavailable states.
Plus long ago I have reported @andybraren and ju regarding the unavailable states of data resiliency

Reply:
I'm not very familiar with this card's design - @yuvalgalanti let me know if I'm missing anything in my suggestions below.

1 - Based on the original mockup it looks like the title size and position was intended to be the same as the nearby Capacity card's title.

![image](https://user-images.githubusercontent.com/9122899/62165271-0a849d80-b2ec-11e9-9d80-7a3f43a085d8.png)

Something like:

```
.co-capacity-card__item-title {
    margin-top: 0;
    font-weight: 500; <--
    font-size: 16px;  <--
}
```

2 - For error/unavailable states, I agree @rawagner moving the text around depending on its state isn't great. Sorry for not identifying this problem sooner @afreen23.

The tricky thing is that other unavailable states in these dashboards (charts, graphs, etc.) put text below the icon, but if we do that here then the text will "move" and look odd next to the Capacity card.

It seems like we'll need to compromise for now until this problem is fixed permanently in our 4.3 designs (which will replace the need for this mostly-empty card completely).

Here's what I'd propose. @yuvalgalanti feel free to 👍 or 👎:

"Not available" icon color: `#72767B` to match the color of `EmptyGraph`

![Artboard](https://user-images.githubusercontent.com/9122899/62168645-ff357000-b2f3-11e9-863f-ded0e1056a85.png)

It's not perfect, and I can suggest some spacing-related CSS for this card and Capacity later, but I would be okay with this general approach for now. Thoughts?